modules/core/include/opencv2/core/mat.inl.hpp

```diff
@@ -85,6 +85,10 @@ template<typename _Tp> inline
 _InputArray::_InputArray(const std::vector<std::vector<_Tp> >& vec)
 { init(FIXED_TYPE + STD_VECTOR_VECTOR + DataType<_Tp>::type + ACCESS_READ, &vec); }
 
+inline
+_InputArray::_InputArray(const std::vector<std::vector<bool> >&)
+{ CV_Error(Error::StsUnsupportedFormat, "std::vector<std::vector<bool> > is not supported!\n"); }
```

Comment:
Thank you for the reivew.

I find that `_OutputArray` uses `CV_Error` to handle `std::vector<bool>` (https://github.com/opencv/opencv/blob/3.2.0/modules/core/include/opencv2/core/mat.inl.hpp#L154). Hence `CV_Error` is used here to keep it consistent.